src/js/control/textarea.tinymce.js

```diff
@@ -78,13 +78,24 @@ export default class controlTinymce extends controlTextarea {
     // define options & allow them to be overwritten in the class config
     const options = jQuery.extend(this.editorOptions, this.classConfig)
     options.target = this.field
-    // initialise the editor
-    window.tinymce.init(options)
+
+    setTimeout(() => {
+      // initialise the editor
+      window.tinymce.init(options)
+    }, 100)
 
     // Set userData
     if (this.config.userData) {
       window.tinymce.editors[this.id].setContent(this.parsedHtml(this.config.userData[0]))
     }
+
+    if (window.lastFormBuilderCopiedTinyMCE) {
+      setTimeout(() => {
+        window.tinymce.editors[this.id].setContent(this.parsedHtml(window.lastFormBuilderCopiedTinyMCE))
+        window.lastFormBuilderCopiedTinyMCE = null
+      }, 300)
```

Comment:
Let's clear the timeout in the callback

```suggestion
      const timeout = setTimeout(() => {
        window.tinymce.editors[this.id].setContent(this.parsedHtml(window.lastFormBuilderCopiedTinyMCE))
        window.lastFormBuilderCopiedTinyMCE = null
        clearTimeout(timeout)
      }, 300)
```